pom.xml

```diff
@@ -22,9 +22,11 @@
   <properties>
     <revision>1.36</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.3</jenkins.version>
+    <jenkins.version>2.218-rc29200.f79828ffaafc</jenkins.version>
     <java.level>8</java.level>
     <tagNameFormat>configuration-as-code-@{project.version}</tagNameFormat>
+    <!-- Because of the new base line -->
+    <slf4jVersion>1.7.26</slf4jVersion>
```

Comment:
better to just bump the plugin parent pom I think

Reply:
I saw that the latest non-beta release didn't allow us to do it: https://github.com/jenkinsci/plugin-pom/blob/33efb037d26230c1d238a7b4977b7f53c29a27bb/pom.xml#L801, so I prefered to keep it as it instead of bumping to a beta version, but I'm happy to bump the parent pom if you prefer that.

Reply:
beta should be fine for pom. good to get testing on it,
I bumped it in a couple of 2.191 bumps recently and it fixed a couple of conflicts